content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -100,32 +84,63 @@ Each HTTP rule contains the following information:
 * An optional host. In this example, no host is specified, so the rule applies to all inbound
   HTTP traffic through the IP address specified. If a host is provided (for example,
   foo.bar.com), the rules apply to that host.
-* A list of paths (for example, `/testpath`), each of which has an associated backend defined with a `serviceName`
-  and `servicePort`. Both the host and path must match the content of an incoming request before the
+* A list of paths (for example, `/testpath`), each of which has an associated backend defined with a `service.Name`
+  and a `service.Port.Name` or `service.Port.Number`. Both the host and path must match the content of an incoming request before the
   load balancer directs traffic to the referenced Service.
 * A backend is a combination of Service and port names as described in the
-  [Service doc](/docs/concepts/services-networking/service/). HTTP (and HTTPS) requests to the
+  [Service doc](/docs/concepts/services-networking/service/) or a [custom resource backend](#resource-backends) by way of a CRD. HTTP (and HTTPS) requests to the
   Ingress that matches the host and path of the rule are sent to the listed backend.
 
-A default backend is often configured in an Ingress controller to service any requests that do not
+A `defaultBackend` is often configured in an Ingress controller to service any requests that do not
 match a path in the spec.
 
-### Default Backend
+### DefaultBackend {#default-backend}
 
-An Ingress with no rules sends all traffic to a single default backend. The default
-backend is typically a configuration option of the [Ingress controller](/docs/concepts/services-networking/ingress-controllers) and is not specified in your Ingress resources.
+An Ingress with no rules sends all traffic to a single default backend. The `defaultBackend` is conventionally a configuration option
+of the [Ingress controller](/docs/concepts/services-networking/ingress-controllers) and is not specified in your Ingress resources.
 
 If none of the hosts or paths match the HTTP request in the Ingress objects, the traffic is
 routed to your default backend.
 
+### Resource Backends {#resource-backend}
+
+A _`Resource`_ backend is an ObjectRef to another Kubernetes resource within the
```

Comment:
We usually don't overly apply font styles. Please consider remove the "_" before and after the word.

Reply:
Updated, thanks!